scripts/zones/Bastok_Markets/Zone.lua

```diff
@@ -18,7 +18,8 @@ zoneObject.onZoneIn = function(player, prevZone)
     -- FIRST LOGIN (START CS)
     if player:getPlaytime(false) == 0 then
         if xi.settings.main.NEW_CHARACTER_CUTSCENE == 1 then
-            cs = 0
+            ---@diagnostic disable-next-line: cast-local-type
+            cs = { 0, -1, 0x93 }
```

Comment:
Overall, the PR looks good.

Those comments could be changed to be a little more descriptive of what those values are.
-- { cs id, param?, ? }

Reply:
Well, I meant the
`---@diagnostic disable-next-line: cast-local-type`

I honestly don't know what that is

Reply:
I'm not a fan of disabling checks if we can find a better way.  Should we just default to a table in all zone.lua?

Reply:
I've switched to this, its probably better that we do just start converting to tables as needed. Since i'm sure we will find most cutscenes have some sort of flag that we are missing.